libs/ui/src/components/NeoCollapse/NeoCollapseStack.vue

```diff
@@ -0,0 +1,11 @@
+<template>
```

Comment:
sorry @prachi00  I'm still not convinced this is needed

this is not a component, it just css

I still think this should be in a stylesheet belonging to massmint

Reply:
cool, I deleted that for now, if the usecase arrive we can decide then